config/translations.cmake

```diff
@@ -0,0 +1,35 @@
+# Translation files
+set(SQLB_TSS
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_ar_SA.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_cs.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_zh.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_zh_TW.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_de.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_es_ES.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_fr.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_ru.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_pl.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_pt_BR.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_en_GB.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_ko_KR.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_tr.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_uk_UA.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_it.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_ja.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_nl.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_sv.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_id.ts"
+    "${CMAKE_SOURCE_DIR}/src/translations/sqlb_ro.ts"
```

Comment:
Is this having advantage of this integration: https://doc.qt.io/qt-6/cmake-commands-qtlinguisttools.html ? See https://github.com/sqlitebrowser/sqlitebrowser/discussions/3451#discussioncomment-6937104

By the way, which is the order of this list? Maybe chronological, and it would make sense to change it to an alphabetical sorting.

Reply:
I don't know the minimum Qt5 version we need to support. I have compiled it with Qt 5.12.2 on Windows.
The order is taken directly from the old config.

Reply:
The list is sorted